src/goto-programs/slice_global_inits.cpp

```diff
@@ -61,22 +61,17 @@ void slice_global_inits(goto_modelt &goto_model)
     if(it == goto_functions.function_map.end())
       continue;
 
-    const goto_programt &goto_program = it->second.body;
-
-    forall_goto_program_instructions(i_it, goto_program)
-    {
-      const codet &code = i_it->code;
-      find_symbols(code, symbols, true, false);
-
-      if(i_it->has_condition())
-        find_symbols(i_it->get_condition(), symbols, true, false);
-    }
+    for(const auto &i : it->second.body.instructions)
+      i.apply([&symbols](const exprt &expr) {
```

Comment:
This for loop should have braces because its body is spread over multiple lines.